src/domain-registry-essentials/cns-smart-contracts.md

```diff
@@ -18,58 +18,107 @@ This section lists all the smart contracts that users can directly interact with
 
 `Registry` is the central smart contract, which stores all CNS domains. Implementing the ERC-721 non-fungible token standard, `Registry` defines domain ownership rules. It stores owner and `Resolver` addresses. For more details, see [Architecture overview - Registry](architecture-overview.md#registry).
 
-| Network | Contract address |
-| :--- | :--- |
-| Mainnet | [0xD1E5b0FF1287aA9f9A268759062E4Ab08b9Dacbe](https://etherscan.io/address/0xd1e5b0ff1287aa9f9a268759062e4ab08b9dacbe) |
+<table>
+    <th>Network</th>
+    <th>Contract address</th>
+    <tr>
+        <td>Mainnet</td>
+        <td><a href="https://etherscan.io/address/0xD1E5b0FF1287aA9f9A268759062E4Ab08b9Dacbe">0xD1E5b0FF1287aA9f9A268759062E4Ab08b9Dacbe</a></td>
+    </tr>
+    <tr>
+        <td>Rinkeby</td>
+        <td><a href="https://etherscan.io/address/0xAad76bea7CFEc82927239415BB18D2e93518ecBB">0xAad76bea7CFEc82927239415BB18D2e93518ecBB</a></td>
+    </tr>
+ </table>
 
 **Source code:** [contracts/Registry.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/Registry.sol)
 
 ### Resolver
 
 `Resolver` is the smart contract that stores domain records and provides methods for domain resolution. For more details, see [Architecture overview - Resolver](architecture-overview.md#resolver).
 
-| Network | Contract address |
-| :--- | :--- |
-| Mainnet | [0xb66DcE2DA6afAAa98F2013446dBCB0f4B0ab2842](https://etherscan.io/address/0xb66DcE2DA6afAAa98F2013446dBCB0f4B0ab2842) |
-
-**Legacy Mainnet Resolvers:**
-
-* [0xa1cac442be6673c49f8e74ffc7c4fd746f3cbd0d](https://etherscan.io/address/0xa1cac442be6673c49f8e74ffc7c4fd746f3cbd0d)
-* [0x878bc2f3f717766ab69c0a5f9a6144931e61aed3](https://etherscan.io/address/0x878bc2f3f717766ab69c0a5f9a6144931e61aed3)
+<table>
+    <th>Network</th>
+    <th>Contract address</th>
+    <th>Legacy addresses</th>
+    <tr>
+        <td>Mainnet</td>
+        <td><a href="https://etherscan.io/address/0xb66DcE2DA6afAAa98F2013446dBCB0f4B0ab2842">0xb66DcE2DA6afAAa98F2013446dBCB0f4B0ab2842</a></td>
+        <td>
+            <a href="https://etherscan.io/address/0xa1cac442be6673c49f8e74ffc7c4fd746f3cbd0d">0xa1cac442be6673c49f8e74ffc7c4fd746f3cbd0d</a>,
+            <a href="https://etherscan.io/address/0x878bc2f3f717766ab69c0a5f9a6144931e61aed3">0x878bc2f3f717766ab69c0a5f9a6144931e61aed3</a>
+        </td>
+    </tr>
+    <tr>
+        <td>Rinkeby</td>
+        <td><a href="https://etherscan.io/address/0x95AE1515367aa64C462c71e87157771165B1287A">0x95AE1515367aa64C462c71e87157771165B1287A</a></td>
+        <td>
+        </td>
+    </tr>
+ </table>
```

Comment:
I'd add a separate table for legacy addresses, this scrollbar looks a bit ugly

![image](https://user-images.githubusercontent.com/2943388/98828779-278da300-2441-11eb-9c77-83fc7587325f.png)

Reply:
Agree, check out renders one more time. Looks better.

Reply:
Also updated alignment for Legacy addresses table. 
![image](https://user-images.githubusercontent.com/38461171/98947773-49009480-24fe-11eb-84f4-4bee5effd428.png)